src/api/requests/ProposalUpdateRequest.ts

```diff
@@ -12,10 +12,6 @@ export class ProposalUpdateRequest extends RequestBody {
 
     @IsNotEmpty()
     public submitter: string;
-    @IsNotEmpty()
-    public blockStart: number;
-    @IsNotEmpty()
-    public blockEnd: number;
```

Comment:
add timeStart, timeEnd, postedAt, receivedAt, expiredAt
all @IsNotEmpty()

Reply:
See above. Why timeStart/timeEnd if we have postedAt/expiredAt? Same thing